### DISCUSSION TITLE
Add Your Work to Simulately

### DISCUSSION THREAD

**Discussion by YuyangLee** [geng-haoran/Simulately#2] [General]
# Add Your Work to the Related Work Page in Simulately

We maintain a list of [Related Work](https://simulately.wiki/related/) on [Simulately](https://simulately.wiki/). This includes awesome papers, benchmarks, workshops, *e.t.c.*, concerning robotics and simulation. If you like it, you can add your work to it via this discussion!

## Requirements

Your work should follow the following requirements:

- It could be a finished project, a published paper (even on Arxiv), rather than a work-in-progress one with empty pages.
- It should be in the field of 3D computer vision or robotics, with real content.
- It should have a stable link (a Netlify/Vercel deploy preview is not allowed).

To request your work to be presented on the website, you only need to post a comment below, and include the following information clearly:

- **Title**
- **A description to your paper**, ≤120 characters
- **Link to your website or the PDF**
- **Which simulator is used?**
- **Which topic is related?** You can follow the ones existing in the [Page](https://simulately.wiki/related/), or you can also add yours.
- (Optional) **A link to your teaser.** An image with proper file size (<2MiB) is preferred.

To ensure our related work page remains usable over time, we reserve the right to reject some lower-quality sites.

## After Your Request

We'll get your site added as soon as possible. In case your request is invalid, we will tell you.

## Notes

- You can ask for an update on your work anytime!
- We keep the right to edit your submission before publication.
- After the update is completed, we will delete your comment.

> This page is motivated by [this discussion](https://github.com/facebook/docusaurus/discussions/7826) for Docusaurus.

**Comment by btx0424** [comment]
Hi there, your efforts have been amazing.

I would like to add another piece to the picture: simulated environments for drones.

Our work [OmniDrones](https://arxiv.org/abs/2309.12825) (submitted to RA-L) presents an efficient and versatile simulated environment for reinforcement learning of UAV control and coordination. It is developed based on *Isaac Sim* and aimed to serve as a starting point for subsequent research. Compared to previous works, it allows users to build more complex scenarios (e.g., a group of drones carrying a payload) where the potential of RL is yet to be fully revealed.

It should fit in the category of [domain-specific simulators](https://simulately.wiki/docs/category/domain-specific-simulators).

[Repo](https://github.com/btx0424/OmniDrones) | [Website](https://omnidrones.readthedocs.io/en/latest/)

**Reply by geng-haoran** [reply]
Hi @btx0424, thanks for the discussion and this great work. We will add this project somewhere in [domain-specific simulators](https://simulately.wiki/docs/category/domain-specific-simulators). Thank you again.

**Comment by tongzhoumu** [comment]
Hi there,

Thanks for the great work! 

I found the link to our work [ManiSkill](https://github.com/haosulab/ManiSkill) is wrong, could you fix it?
<img width="844" alt="image" src="https://github.com/geng-haoran/Simulately/assets/8188853/86dcc8bc-bb3b-44fa-b619-3095aa4d7c7e">

- Correct link: https://github.com/haosulab/ManiSkill
- Venue: NeurIPS 2021 (Dataset and Benchmark Track)

Thanks!

**Reply by geng-haoran** [reply]
Thank you! We will fix it soon!

**Reply by geng-haoran** [reply]
Already updated! Thank you again!
